index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+
+    <title>Portfolio setup and mobile version skeleton</title>
+</head>
+<body>
+
+<header>
+
+  <nav>
+    <ul>
+     <li id="logo"><h1>Welcome</h1></li>
```

Comment:
- [ ] Kindly change your logo into a link as stated in the project requirement.
> The logo (if the design has one) is on the left and has a link (`a` tag)

Reply:
Thanks for the review. I had a link prior to this second review but the reviewer suggested I used an hx header instead.